index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+	<head>
+		<meta charset="utf-8">
+		<title>Apple</title>
+
+		<link rel="stylesheet" type="text/css" href="css/reset.css">
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+	</head>
+	<body>
+		<main class="main-container">
+			<div class="container">
+				<nav class="nav-menu">
+					<div class="menu">
+						<button class="menu-link" id="logo" type="button" name="button"><a href="https://web.archive.org/web/20140301004610/http://store.apple.com/"><img src="img/apple.png" alt="Logo" width="20px"></a></button>
+						<button class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://store.apple.com/">Store</a></button>
+						<button class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/mac/">Mac</a></button>
+						<button class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipod/">iPod</a></button>
+						<button class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone/">iPhone</a></button>
+						<button class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipad/">iPad</a></button>
+						<button class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/itunes/">iTunes</a></button>
+						<button class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/support/">Support</a></button>
+						<button class="menu-link" id="button"><input class="search" type="search" name="search"></li>
+					</div>
+				</nav>
```

Comment:
The closing tag of `button` for the input is missing, you can install html linter in your preferred code editor, and it will discover these issues for you.